flask_appbuilder/templates/appbuilder/general/security/login_pam.html

```diff
@@ -0,0 +1,62 @@
+<!-- extend base layout -->
+{% extends "appbuilder/base.html" %}
+
+
+
+{% block content %}
+
+<div class="container">
+    <div id="loginbox" style="margin-top:50px;" class="mainbox col-md-6 col-md-offset-3 col-sm-8 col-sm-offset-2">
+        <div class="panel panel-primary" >
+            <div class="panel-heading">
+                <div class="panel-title">{{ title }}</div>
+            </div>
+            <div style="padding-top:30px" class="panel-body" >
+
+                <form class="form" action="" method="post" name="login">
+                    {{form.hidden_tag()}}
+                    <div class="help-block">{{_("Enter your login and password below")}}:</div>
+                    <div class="control-group{% if form.errors.openid is defined %} error{% endif %}">
+                        <label class="control-label" for="openid">{{_("Username")}}:</label>
+
+                        <div class="controls">
+                            <div class="input-group">
+                                <span class="input-group-addon"><i class="fa fa-user"></i></span>
+                                {{ form.username(size = 80, class = "form-control", autofocus = true) }}
+                            </div>
+                            {% for error in form.errors.get('openid', []) %}
```

Comment:
?

Reply:
This is here for showing any errors. I have used same login page content as for DB authentication.